readme.md

```diff
@@ -65,12 +65,12 @@ Returns a `FSM` object, the `FSM` represents a Deterministic Finite Automata(DFA
 ###Usage in Node.js
 
 ```
-var regParser = require('regparser');
+var regParser = require('reg2automata');
 
```

Comment:
Thanks for the PR!
Instead of `regParser`, I think `reg2automata` is a better name. Could you please rename it?
Also please update the sample code in `Usage in Browser` for keeping consistence.